shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/java/org/apache/shardingsphere/sql/parser/integrate/asserts/statement/dml/impl/SelectStatementAssert.java

```diff
@@ -70,10 +68,6 @@ private static void assertProjection(final SQLCaseAssertContext assertContext, f
         ProjectionAssert.assertIs(assertContext, actual.getProjections(), expected.getProjections());
     }
     
-    private static void assertTable(final SQLCaseAssertContext assertContext, final SelectStatement actual, final SelectStatementTestCase expected) {
-        TableAssert.assertIs(assertContext, actual.getSimpleTableSegments(), new LinkedList<>(expected.getTables().getSimpleTables()));
```

Comment:
Why do you need to remove them?

Reply:
Because it was changed to the test of tablereference, this function is not used。